src/validators.js

```diff
@@ -1,18 +1,21 @@
 import { base } from '@tokend/js-sdk'
 
 import * as validators from 'vuelidate/lib/validators'
+import { validateAddress } from '@/js/helpers/address-validation'
```

Comment:
let's declare the validator function right here.

having `/helpers` directory isn't good at all, before putting smth to `/helpers` `/common`, `/utils` or so, try to avoid it first

Reply:
ok